packages/server/src/sdk/app/tables/internal/sqs.ts

```diff
@@ -107,8 +108,18 @@ async function buildBaseDefinition(): Promise<PreSaveSQLiteDefinition> {
 
 export async function syncDefinition(): Promise<void> {
   const db = context.getAppDB()
+  let existing: SQLiteDefinition | undefined
+  if (await db.exists(SQLITE_DESIGN_DOC_ID)) {
+    existing = await db.get<SQLiteDefinition>(SQLITE_DESIGN_DOC_ID)
+  }
```

Comment:
Could remove a DB call by wrapping just a `db.get` in a try-catch. Not sure how often this function is going to run in practice to know whether that's worth doing.